docs/pages/database-access/guides/redis.mdx

```diff
@@ -0,0 +1,359 @@
+    * `server.key` with generated private key
+    * `server.crt` with generated user certificate.
+    You will need these files to enable mutual TLS on your Redis server.
+  </TabItem>
+  <TabItem label="Cluster">
+    When connecting to Redis Cluster, sign certificates for each member
+    using their hostnames and IP addresses.
+
+    For example, if the first member is accessible at `redis1.example.com` with IP `10.0.1` and
+    the second at `redis2.example.com` with IP `10.0.0.2`, run:
+
+    ```code
+    $ tctl auth sign --format=redis --host=redis1.example.com,10.0.0.1 --out=redis1 --ttl=2190h
+    $ tctl auth sign --format=redis --host=redis2.example.com,10.0.0.2 --out=redis2 --ttl=2190h
+    ```
+
+    (!docs/pages/includes/database-access/ttl-note.mdx!)
+
+    The command will create three files:
+    * `server.cas` with Teleport's certificate authority
+    * `server.key` with generated private key
+    * `server.crt` with generated user certificate.
+    You will need these files to enable mutual TLS on your Redis server.
+  </TabItem>
+</Tabs>
+
+(!docs/pages/includes/database-access/rotation-note.mdx!)
+
+Use the generated secrets to enable mutual TLS in your `redis.conf` configuration
+file and restart the database:
+
+<Tabs>
+  <TabItem label="Standalone">
+    ```ini
+    tls-port 6379
+    port 0
+
+    aclfile /path/to/users.acl
+
+    tls-ca-cert-file /path/to/server.cas
+    tls-cert-file /path/to/server.crt
+    tls-key-file /path/to/server.key
+    tls-protocols "TLSv1.2 TLSv1.3"
+    ```
+  </TabItem>
+  <TabItem label="Cluster">
+    ```ini
+    tls-port 7001
+    port 0
+
+    cluster-enabled yes
+
+    tls-replication yes
+    tls-cluster yes
+
+    aclfile /path/to/users.acl
+
+    masterauth GENERATED_STRONG_PASSWORD
+    masteruser replica-user
+
+    tls-cert-file /usr/local/etc/redis/certs/server.crt
+    tls-key-file /usr/local/etc/redis/certs/server.key
+    tls-ca-cert-file /usr/local/etc/redis/certs/server.cas
+    tls-protocols "TLSv1.2 TLSv1.3"
+    ```
+  </TabItem>
+</Tabs>
+
+Once mutual TLS has been enabled, you will no longer be able to connect to
+the cluster without providing a valid client certificate. You can use the
+`tls-auth-clients optional` setting to allow connections
+from clients that do not present a certificate.
+
+See [TLS Support](https://redis.io/topics/encryption)
+in the Redis documentation for more details.
+
+<Admonition type="note" title="Create Redis Cluster">
+  When you're configuring Redis Cluster you need to create the cluster using `redis-cli`.
+
+  Use the following command to create the cluster. Please note `redis-cli --cluster create` accepts only IP addresses.
+
+  ```shell
+  export REDISCLI_AUTH=STRONG_GENERATED_PASSWORD
+  export CERTS_DIR=/path/to/certs/
+
+  redis-cli --user alice --cluster-replicas 1 --tls --cluster-yes \
+  --cluster create ${IP1}:7001 ${IP2}:7002 ${IP3}:7003 ${IP4}:7004 ${IP5}:7005 ${IP6}:7006 \
```

Comment:
I agree, but I've got no better idea rather than add a note with something like "export your Redis IPs as environment variables IP1...IP6". Do you think that would help or for example using some "fake" IPs in this case make more sense?